lib/que/locker.rb

```diff
@@ -215,10 +216,16 @@ def initialize(
             @connection.wrapped_connection.close
           end
         end
+
+      @pidfile = pidfile
+      write_pid
+      at_exit { delete_pid } # Guaranty delete the pidfile on exit or crashed.
```

Comment:
I think setting up this deletion before writing the file would be ever so slightly more robust - move this line up one, I reckon. And I don't think the comment is necessary - it's pretty clear what this is for.